### ISSUE TITLE
stephen_01.smt2 + --lp-icp option

### ISSUE THREAD

**Issue by soonhokong** [dreal/dreal3#265]
```bash
$ ./dReal ../../src/tests/nra/stephen_01.smt2 --lp-icp
terminate called after throwing an instance of 'std::runtime_error'
  what():  GLPK simplex failed: GLP_EFAIL
[1]    41467 abort (core dumped)  ./dReal ../../src/tests/nra/stephen_01.smt2 --lp-icp
```

Stacktrace:
```
#5  0x00007ffff7915288 in __cxa_throw () from /usr/lib/x86_64-linux-gnu/libstdc++.so.6
#6  0x0000000000a20957 in dreal::glpk_wrapper::is_sat (this=0x7fffffffda60) at /usr0/home/soonhok/work/dreal3/src/util/glpk_wrapper.cpp:182
#7  0x0000000000aa8e46 in dreal::lp_icp::is_lp_sat (lp_solver=..., solution=..., config=...) at /usr0/home/soonhok/work/dreal3/src/icp/lp_icp.cpp:81
#8  0x0000000000aa98c7 in dreal::lp_icp::solve (ctc=..., cs=..., constraints=..., brancher=...) at /usr0/home/soonhok/work/dreal3/src/icp/lp_icp.cpp:162
#9  0x00000000009efb95 in dreal::nra_solver::check (this=0x14e8980, complete=true) at /usr0/home/soonhok/work/dreal3/src/dsolvers/nra_solver.cpp:492
```

**Comment by soonhokong** [comment]
Closed by 55335dadba693e60c493b602e06e73d996121c59

**Comment by soonhokong** [comment]
Thanks @dzufferey!
